apollo-router/tests/integration/mod.rs

```diff
@@ -1,8 +1,6 @@
 #[path = "../common.rs"]
 pub(crate) mod common;
 pub(crate) use common::IntegrationTest;
-pub(crate) use common::Telemetry;
-pub(crate) use common::ValueExt;
```

Comment:
I get lint errors locally about these being unused, but they seem to be causing lint to fail in CI. Any idea what the discrepancy is? Going to revert this change.

Reply:
Are you running `cargo task lint`?

Reply:
@garypen discovered that the tests are only enabled on linux targets, so the includes here seem unused on our MacOS development machines.

Reply:
We'll fix this in a separate PR.